zerver/lib/integrations.py

```diff
@@ -315,6 +315,7 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
     ),
     WebhookIntegration('gitlab', ['version-control'], display_name='GitLab'),
     WebhookIntegration('gogs', ['version-control'], stream_name='commits'),
+    WebhookIntegration('gocd', ['continuous-integration'], display_name='Gocd'),
```

Comment:
Change `display_name` to `GoCD`, which is how the name is actually capitalised. :)